flink-ai-flow/ai_flow/java/client/src/main/java/com/aiflow/client/AIFlowClient.java

```diff
@@ -926,4 +929,167 @@ public void stopListenEvent(String namespace, String key) {
     public long getLatestVersion(String namespace, String key) throws Exception {
         return this.notificationClient.getLatestVersion(namespace, key);
     }
+
+    /**
+     * Register a MetricMeta in metric center.
+     *
+     * @param name          Name of registered metric meta. This is expected to be unique in the backend store.
+     * @param metricType    Type of registered metric meta.
+     * @param projectName   Name of the project associated with the registered metric meta.
+     * @param description   Name of registered model.
+     * @param datasetName   Name of the dataset associated with the registered metric meta.
+     * @param modelName     Name of the model associated with the registered metric meta.
+     * @param jobName       Name of the job associated with the registered metric meta.
+     * @param startTime     Start time of registered metric meta.
+     * @param endTime       End time of registered metric meta.
+     * @param uri           Uri of registered metric meta.
+     * @param tags          Tags of registered metric meta.
+     * @param properties    Properties of registered metric meta.
+     *
+     * @return A single :py:class:`ai_flow.meta.metric_meta.MetricMeta` object.
```

Comment:
Update the return type of `registerMetricMeta`.

Reply:
Corrected